utility/importer/brave_importer.cc

```diff
@@ -122,6 +122,13 @@ void BraveImporter::StartImport(const importer::SourceProfile& source_profile,
   bridge_->NotifyEnded();
 }
 
+// Called before user-toggleable import items.
+// These import types don't need a distinct checkbox in the import screen.
+void BraveImporter::ImportRequiredItems() {
+  ImportReferral();
+  ImportSettings();
```

Comment:
why don't you use importer::SEARCH_ENGINES for search engine importing?

Reply:
I think it should always be imported; but we could make a UI flag for it. I asked @rebron @davidtemkin and @bbondy on Slack (waiting for response)

Reply:
it is different between always be imported and checkbox with default checked.
Both achieve the same goal but the later one allows user to opt out

Reply:
This should be good for now - per convo in Slack
https://bravesoftware.slack.com/archives/C70UTN17S/p1544637587376900